app/experimenter/experiments/tests/test_forms.py

```diff
@@ -1062,6 +1065,40 @@ def test_form_saves_objectives(self):
 
         self.assertEqual(experiment.objectives, data["objectives"])
         self.assertEqual(experiment.analysis, data["analysis"])
+        self.assertTrue(experiment.survey)
+        self.assertEqual(experiment.survey_urls, data["survey_urls"])
+        self.assertEqual(
+            experiment.survey_instructions, data["survey_instructions"]
+        )
+
+    def test_removes_survey_urls_and_instructions_if_changed_to_false(self):
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT,
+            survey=True,
+            survey_urls="example.com",
+            survey_instructions="Here are the survey instructions.",
+        )
+
+        data = {
+            "objectives": "The objective is to experiment!",
+            "analysis_owner": "Jim Bob The Data Scientist",
+            "analysis": "Lets analyze the results!",
+            "survey": False,
+            "survey_urls": "example.com",
+            "survey_instructions": "Here are the survey instructions.",
+        }
+
+        form = ExperimentObjectivesForm(
+            request=self.request, data=data, instance=experiment
+        )
+
+        self.assertTrue(form.is_valid())
+
+        experiment = form.save()
+
+        self.assertFalse(experiment.survey)
+        self.assertFalse(experiment.survey_urls)
+        self.assertFalse(experiment.survey_instructions)
```

Comment:
Should these be false, or empty string? (Are you taking advantage of Python falsiness here? I would probably be explicit.)